src/test/java/io/dockstore/provision/DOSPluginUnitTest.java

```diff
@@ -88,6 +135,134 @@ public void testPrepareDownload() throws IOException {
         while ((line = in.readLine()) != null) {
             content.append(line);
         }
+        in.close();
```

Comment:
In the future, watch out. It doesn't really matter here, but there are cases where this won't execute this way 
(for example if https://docs.oracle.com/javase/7/docs/api/java/io/BufferedReader.html#readLine() throws an exception)